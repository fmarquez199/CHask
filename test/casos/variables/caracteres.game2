Rune a = *a*
Rune b = *b*
Rune c = *c*
Rune d = *d*
Rune e = *e*
Rune f = *f*
Rune g = *g*
Rune h = *h*
Rune i = *i*
Rune j = *j*
Rune k = *k*
Rune l = *l*
Rune m = *m*
Rune n = *n*
Rune o = *o*
Rune p = *p*
Rune q = *q*
Rune r = *r*
Rune s = *s*
Rune t = *t*
Rune u = *u*
Rune v = *v*
Rune w = *w*
Rune x = *x*
Rune y = *y*
Rune z = *z*
Rune A = *A*
Rune B = *B*
Rune C = *C*
Rune D = *D*
Rune E = *E*
Rune F = *F*
Rune G = *G*
Rune H = *H*
Rune I = *I*
Rune J = *J*
Rune K = *K*
Rune L = *L*
Rune M = *M*
Rune N = *N*
Rune O = *O*
Rune P = *P*
Rune Q = *Q*
Rune R = *R*
Rune S = *S*
Rune T = *T*
Rune U = *U*
Rune V = *V*
Rune W = *W*
Rune X = *X*
Rune Y = *Y*
Rune Z = *Z*
Rune n0 = *0*
Rune n1 = *1*
Rune n2 = *2*
Rune n3 = *3*
Rune n4 = *4*
Rune n5 = *5*
Rune n6 = *6*
Rune n7 = *7*
Rune n8 = *8*
Rune n9 = *9*
Rune sp1 = *!*
Rune sp2 = *"*
Rune sp3 = *#*
Rune sp4 = *$*
Rune sp5 = *%*
Rune sp6 = *&*
Rune sp7 = *'*
Rune sp8 = *(*
Rune sp9 = *)*
Rune sp10 = *\**
Rune sp11 = *+*
Rune sp12 = *,*
Rune sp13 = *-*
Rune sp14 = *.*
Rune sp15 = */*
Rune sp16 = *:*
Rune sp17 = *;*
Rune sp18 = *<*
Rune sp19 = *=*
Rune sp20 = *>*
Rune sp21 = *?*
Rune sp22 = *@*
Rune sp23 = *[*
Rune sp24 = *\*
Rune sp25 = *]*
Rune sp26 = *^*
Rune sp27 = *_*
Rune sp28 = *`*
Rune sp29 = *{*
Rune sp30 = *|*
Rune sp31 = *}*
Rune sp32 = *~*
