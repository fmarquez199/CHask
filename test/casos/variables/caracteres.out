Token Rune en la fila: 1, columna: 1
Token caracter *a* en la fila: 2, columna: 1
Token caracter *b* en la fila: 3, columna: 1
Token caracter *c* en la fila: 4, columna: 1
Token caracter *d* en la fila: 5, columna: 1
Token caracter *e* en la fila: 6, columna: 1
Token caracter *f* en la fila: 7, columna: 1
Token caracter *g* en la fila: 8, columna: 1
Token caracter *h* en la fila: 9, columna: 1
Token caracter *i* en la fila: 10, columna: 1
Token caracter *j* en la fila: 11, columna: 1
Token caracter *k* en la fila: 12, columna: 1
Token caracter *l* en la fila: 13, columna: 1
Token caracter *m* en la fila: 14, columna: 1
Token caracter *n* en la fila: 15, columna: 1
Token caracter *o* en la fila: 16, columna: 1
Token caracter *p* en la fila: 17, columna: 1
Token caracter *q* en la fila: 18, columna: 1
Token caracter *r* en la fila: 19, columna: 1
Token caracter *s* en la fila: 20, columna: 1
Token caracter *t* en la fila: 21, columna: 1
Token caracter *u* en la fila: 22, columna: 1
Token caracter *v* en la fila: 23, columna: 1
Token caracter *w* en la fila: 24, columna: 1
Token caracter *x* en la fila: 25, columna: 1
Token caracter *y* en la fila: 26, columna: 1
Token caracter *z* en la fila: 27, columna: 1
Token caracter *A* en la fila: 28, columna: 1
Token caracter *B* en la fila: 29, columna: 1
Token caracter *C* en la fila: 30, columna: 1
Token caracter *D* en la fila: 31, columna: 1
Token caracter *E* en la fila: 32, columna: 1
Token caracter *F* en la fila: 33, columna: 1
Token caracter *G* en la fila: 34, columna: 1
Token caracter *H* en la fila: 35, columna: 1
Token caracter *I* en la fila: 36, columna: 1
Token caracter *J* en la fila: 37, columna: 1
Token caracter *K* en la fila: 38, columna: 1
Token caracter *L* en la fila: 39, columna: 1
Token caracter *M* en la fila: 40, columna: 1
Token caracter *N* en la fila: 41, columna: 1
Token caracter *O* en la fila: 42, columna: 1
Token caracter *P* en la fila: 43, columna: 1
Token caracter *Q* en la fila: 44, columna: 1
Token caracter *R* en la fila: 45, columna: 1
Token caracter *S* en la fila: 46, columna: 1
Token caracter *T* en la fila: 47, columna: 1
Token caracter *U* en la fila: 48, columna: 1
Token caracter *V* en la fila: 49, columna: 1
Token caracter *W* en la fila: 50, columna: 1
Token caracter *X* en la fila: 51, columna: 1
Token caracter *Y* en la fila: 52, columna: 1
Token caracter *Z* en la fila: 53, columna: 1
Token caracter *0* en la fila: 54, columna: 1
Token caracter *1* en la fila: 55, columna: 1
Token caracter *2* en la fila: 56, columna: 1
Token caracter *3* en la fila: 57, columna: 1
Token caracter *4* en la fila: 58, columna: 1
Token caracter *5* en la fila: 59, columna: 1
Token caracter *6* en la fila: 60, columna: 1
Token caracter *7* en la fila: 61, columna: 1
Token caracter *8* en la fila: 62, columna: 1
Token caracter *9* en la fila: 63, columna: 1
Token caracter *!* en la fila: 64, columna: 1
Token caracter *"* en la fila: 65, columna: 1
Token caracter *#* en la fila: 66, columna: 1
Token caracter *$* en la fila: 67, columna: 1
Token caracter *%* en la fila: 68, columna: 1
Token caracter *&* en la fila: 69, columna: 1
Token caracter *'* en la fila: 70, columna: 1
Token caracter *(* en la fila: 71, columna: 1
Token caracter *)* en la fila: 72, columna: 1
Token caracter *\** en la fila: 73, columna: 1
Token caracter *+* en la fila: 74, columna: 1
Token caracter *,* en la fila: 75, columna: 1
Token caracter *-* en la fila: 76, columna: 1
Token caracter *.* en la fila: 77, columna: 1
Token caracter */* en la fila: 78, columna: 1
Token caracter *:* en la fila: 79, columna: 1
Token caracter *;* en la fila: 80, columna: 1
Token caracter *<* en la fila: 81, columna: 1
Token caracter *=* en la fila: 82, columna: 1
Token caracter *>* en la fila: 83, columna: 1
Token caracter *?* en la fila: 84, columna: 1
Token caracter *@* en la fila: 85, columna: 1
Token caracter *[* en la fila: 86, columna: 1
Token caracter *]* en la fila: 87, columna: 1
Token caracter *^* en la fila: 88, columna: 1
Token caracter *_* en la fila: 89, columna: 1
Token caracter *`* en la fila: 90, columna: 1
Token caracter *{* en la fila: 91, columna: 1
Token caracter *|* en la fila: 92, columna: 1
Token caracter *}* en la fila: 93, columna: 1
Token caracter *\~* en la fila: 94, columna: 1
Token caracter *\0* en la fila: 95, columna: 1
Token caracter *\n* en la fila: 96, columna: 1
Token caracter *\t* en la fila: 97, columna: 1
Token caracter *\\* en la fila: 98, columna: 1
Token caracter *\~* en la fila: 99, columna: 1
Token caracter *}* en la fila: 100, columna: 1

